jicoco/src/main/kotlin/org/jitsi/health/HealthChecker.kt

```diff
@@ -177,6 +177,6 @@ class HealthChecker(
     }
 
     companion object {
-        val stickyFailuresGracePeriodDefault = Duration.ofMinutes(5)
+        val stickyFailuresGracePeriodDefault: Duration = Duration.ofMinutes(5)
```

Comment:
Why this change?

Reply:
My IDE showed a warning, I think Duration.ofMinutes() is not annotated so it doesn't know if it's nullable or not.